dumper/main.py

```diff
@@ -26,27 +29,53 @@ def dot():
     sys.stdout.flush()
 
 
+def dump_forms(query, writer):
+    counter = 0
+    unique_domains = set()
+    for doc in iter_forms_data(es, index, query):
+        counter += 1
+        if counter % 100 == 0:
+            dot()
+        form_obj = parse_form_doc(doc)
+        form_obj.write(writer)
+        unique_domains.add(form_obj.domain)
+
+    write_domains(unique_domains, args.domain_output)
+
+
+def dump_users(query, writer):
+    counter = 0
+    for hit in query_users_data(es, index, query):
+        counter += 1
+        if counter % 100 == 0:
+            dot()
+        user_obj = parse_user(hit)
+        user_obj.write(writer)
+
+
+def execute_query_choice(query_choice):
+    writer = init_writer(query_choice)
+    if 'f' == query_choice:
+        query = get_query('form_dump_query.json')
+        dump_forms(query, writer)
+    elif 'u' == query_choice:
+        query = get_query('user_dump_query.json')
+        dump_users(query, writer)
+    else:
+        raise Exception('Invalid parameter entered for query-type')
+
+
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument("url", help="ES URL")
     parser.add_argument("index", help="ES index")
-    parser.add_argument("query", help="Path to query file")
```

Comment:
Heads up I removed this as a param and am instead selecting the file path based upon the query-type param that I added. I did this for now because it seemed repetitive to have to specify the query type and query file, when we know what query you want to execute based upon the type param. But if we expand this later, may want to revert.